src/coreclr/jit/clrjit.natvis

```diff
@@ -155,6 +155,14 @@ The .NET Foundation licenses this file to you under the MIT license.
     <DisplayString>IG{igNum,d}</DisplayString>
   </Type>
 
+  <Type Name="emitter::instrDesc">
+    <DisplayString Condition="(_idInsFmt == IF_RRD) || (_idInsFmt == IF_RWR) || (_idInsFmt == IF_RRW)">{_idIns,en} {_idReg1,en}</DisplayString>
+    <DisplayString Condition="(_idInsFmt == IF_RRD_CNS) || (_idInsFmt == IF_RWR_CNS) || (_idInsFmt == IF_RRW_CNS)">{_idIns,en} {_idReg1,en}, {_idLargeCns,d}</DisplayString>
+    <DisplayString Condition="(_idInsFmt == IF_RRW_SHF) &amp;&amp; (_idLargeCns != 0)">{_idIns,en} {_idReg1,en}, {_idLargeCns,d}</DisplayString>
+    <DisplayString Condition="(_idInsFmt == IF_RRW_SHF) &amp;&amp; (_idLargeCns == 0)">{_idIns,en} {_idReg1,en}, {_idSmallCns,d}</DisplayString>
+    <DisplayString>{_idIns,en}</DisplayString>
```

Comment:
Could we log a work item to eventually extend this to all `_idInsFmt`? Seems like good "low hanging fruit".

Reply:
Yes - I intentionally left it open ended but I also created https://github.com/dotnet/runtime/issues/53617 and marked it as "up-for-grab".